CHANGES.md

```diff
@@ -1,6 +1,12 @@
 Release Changes List
 ====================
 
+version 1.16.0 pre-release
+==========================
+
+*UNDER CONSTRUCTION: Chapel 1.16.0 is scheduled for release in October 2017.*
+
+
```

Comment:
I wouldn't add this change.  It doesn't really add value, increases the odds that someone will stop reading before getting to the actual valuable content, and is one more thing that will need to be maintained if it's modified now.